src/index.html

```diff
@@ -5,8 +5,254 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+    >
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700;900&display=swap" 
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <img class="header__logo" 
+      src="images/logo.png" alt="logo">
+      <nav class="header__nav">
+        <ul class="menu">
+          <li class="menu__item">
+            <a class="menu__link is-active" href="#">apple</a>
```

Comment:
```suggestion
            <a class="menu__link is-active" href="#">Apple</a>
```
it's always better to write uppercase first